lib/virt_autotest/utils.pm

```diff
@@ -155,9 +157,16 @@ sub create_guest {
         send_key 'ret';    # Make some visual separator
 
         # Run unattended installation for selected guest
-        my $diskformat = get_var("VIRT_QEMU_DISK_FORMAT") // "qcow2";
-        assert_script_run "qemu-img create -f $diskformat /var/lib/libvirt/images/xen/$name.$diskformat 20G", 180;
-        script_run "( virt-install $extra_params --name $name --vcpus=2,maxvcpus=4 --memory=2048,maxmemory=4096 --disk /var/lib/libvirt/images/xen/$name.qcow2 --network network=default,mac=$macaddress --noautoconsole --vnc --autostart --location=$location --wait -1 --events on_reboot=restart --extra-args 'autoyast=" . data_url($autoyast) . "' >> virt-install_$name.txt 2>&1 & )";
+        my ($autoyastURL, $diskformat, $virtinstall);
+        $autoyastURL = data_url($autoyast);
+        $diskformat = get_var("VIRT_QEMU_DISK_FORMAT") // "qcow2";
```

Comment:
Where did `qemu-img create ...` went? Was is removed intentionally?

Reply:
Yes, It was. The `virt-install` is itself capable of creating the disk.
I will try to bring it back tho.